### DISCUSSION TITLE
pass mode doesn't work as expected

### DISCUSSION THREAD

**Discussion by Becheikh** [strongswan/strongswan#2217] [General]
Hi Folks, 
I'm using strongswan v 5.9.6 and when I'm creating bypass policies using **mode= pass**  the policy is created as protect. You can find as attached file  a screenshot showing the policy type. 
I would be grateful if you could tell me what was wrong with my following configuration

connections {
vpp2-trex {
        local_addrs = 192.168.0.1
        remote_addrs = 192.168.0.2
local {

        auth = pubkey
        certs =carolCert.pem
        id= sun.strongswan.org
        }
remote {
        auth = pubkey
        id = moon.strongswan.org
        }
children {
vp2-tr {
        rekey_time = 54
        rekey_bytes = 5000000
        rekey_packets= 10000
        mode = transport
        esp_proposals = aes128gcm128-sha256
        inactivity = 60s
}
drop-eth0-default {
            local_ts  = 0.0.0.0/0
            remote_ts = 0.0.0.0/0
            interface = eth0
            priority = 4

            mode = drop
            start_action = trap
         }
         pass-ssh-in {
            local_ts  = 0.0.0.0/0[tcp/ssh]
            remote_ts = 0.0.0.0/0[tcp]
            priority = 1

            mode = pass
            start_action = trap
         }
         pass-ssh-out {
            local_ts  = 0.0.0.0/0[tcp]
            remote_ts = 0.0.0.0/0[tcp/ssh]
            priority = 1

            mode = pass
            start_action = trap
         }
        pass-http-out {
            local_ts  = 0.0.0.0/0[tcp]
            remote_ts = 192.168.0.150[tcp/http]
            priority = 1
   mode = pass
            start_action = trap
         }
}
version = 2
mobike = no
reauth_time = 10800
proposals = aes128-sha256-x25519

}
}

 
![2024-04-17_16-27](https://github.com/strongswan/strongswan/assets/16711011/8ab43ed6-2de6-466c-937b-0024374cfbf2)

**Comment by tobiasbrunner** [comment]
You don't want to define passthrough policies as child sections of a regular connection. Follow [the examples](https://docs.strongswan.org/docs/5.9/config/quickstart.html#_passthroughbypass_policies).

**Reply by Becheikh** [reply]
It doesn't work, I added the below part to the above  configuration and it failed 
 passthrough-subnet {
      remote_addrs = 127.0.0.1
      children {
        passthrough-subnet {
          local_ts  = 10.1.0.0/24
          remote_ts = 192.168.2.0/24
          mode = pass
          start_action = trap
        }
      }
    }
![2024-04-17_17-32](https://github.com/strongswan/strongswan/assets/16711011/c3e53c4c-4f8e-4db7-a988-80450e4a89ef)
